ftest.sh

```diff
@@ -192,8 +196,12 @@ args+=" $*"
 # shellcheck disable=SC2086
 if ! ssh $SSH_KEY_ARGS ${REMOTE_ACCT:-jenkins}@"${nodes[0]}" "set -ex
 ulimit -c unlimited
-rm -rf $DAOS_BASE/install/tmp
-mkdir -p $DAOS_BASE/install/tmp
+if [ $SL_PREFIX == "/usr" ]; then
```

Comment:
(lint) The double quotes around this do nothing. Remove or escape them. [SC2140]